README.md

```diff
@@ -195,26 +202,26 @@ Object.assign(list, {
 
 ### Data manipulation using `requestReset()`
 
-The `<virtual-list>` element will automatically rerender the displayed items when it receives a new `items` array. For example, to add a new item to the end, you could do:
+The `<virtual-list>` element will automatically rerender the displayed items when `items` changes. For example, to add a new item to the end, you could do:
 
 ```js
-list.items = list.items.concat([{name: 'new item'}]);
+myItems.push('new item');
+list.items++;

```

Comment:
Hmm, having to write that kind of code seems a bit annoying. Really easy to forget the `list.items++`. I hope the data source ideas we talked about make it better.